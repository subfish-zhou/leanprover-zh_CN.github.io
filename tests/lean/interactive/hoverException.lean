inductive Foo
| mk : (a b : Bar) â†’ Foo
            --^ textDocument/hover
