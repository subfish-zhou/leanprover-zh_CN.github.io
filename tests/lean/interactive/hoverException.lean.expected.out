{"textDocument": {"uri": "file://hoverException.lean"},
 "position": {"line": 1, "character": 14}}
null
